### ISSUE TITLE
Every few times visted the page doesnt exist

### ISSUE THREAD

**Issue by afjboulder** [Shazwazza/Articulate#150]
Hi I have noticed that several times in the last few weeks the blog page using Articulate on my site will say the url has moved or doesn't exist. Have you seen this issue/ can you help fix it?

**Comment by Shazwazza** [comment]
Hi, I have no seen this before. All Blog posts are just normal Umbraco pages. Are you sure you are seeing this issue on normal Blog post pages, or are you seeing this on Category or Tag pages?

**Comment by afjboulder** [comment]
Yeah I believe im just calling the main blog list but this is the only page doing it.`<div class="container">
			@Html.Partial("BlogList", new Articulate.Models.ListModel(Umbraco.Content(1647)))
		</div>`

**Comment by Shazwazza** [comment]
I don't understand what the code you've listed above is doing?

Regarding the not found issue, perhaps you have multiple root nodes in Umbraco and haven't assigned domains to them?
